src/storage/disk_table.cc

```diff
@@ -316,10 +389,24 @@ bool DiskTable::Get(uint32_t idx, const std::string& pk, uint64_t ts, std::strin
 bool DiskTable::Get(const std::string& pk, uint64_t ts, std::string& value) { return Get(0, pk, ts, value); }
 
 void DiskTable::SchedGc() {
+    ResetRecordCnt();
     GcHead();
+    GcTTL();
     UpdateTTL();
 }
 
+void DiskTable::ResetRecordCnt() {
```

Comment:
`GetRecordIdxCnt` will return 0 if executing GC ?

Reply:
The RecordIdxCnt will be reset and recounted during GC